Library/Tests/ViewModels/RewardPledgeViewModelTests.swift

```diff
@@ -40,6 +40,7 @@ internal final class RewardPledgeViewModelTests: TestCase {
   fileprivate let differentPaymentMethodButtonHidden = TestObserver<Bool, NoError>()
   fileprivate let dismissViewController = TestObserver<(), NoError>()
   fileprivate let estimatedDeliveryDateLabelText = TestObserver<String, NoError>()
+  private let estimatedFulfillmentStackViewHidden = TestObserver<Bool, NoError>()
```

Comment:
I think the `private`/`fileprivate` RFC may be the most controversial thing that's passed Evolution. I kinda like just using `fileprivate` everywhere to avoid having to know an additional usage pattern with uncertain benefits.